LayoutTests/fast/mediastream/mediaStream-with-rotation-webm.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html>
+<head>
+    <title>MediaRecorder recording video with rotation</title>
+    <script src="../../resources/testharness.js"></script>
+    <script src="../../resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+    async function record(stream, mimeType)
+    {
+        const recorder = new MediaRecorder(stream, { mimeType : mimeType });
+        const promise = new Promise((resolve, reject) => {
+            recorder.ondataavailable = (e) => resolve(e.data);
+            setTimeout(reject, 5000);
+        });
+        recorder.start();
+        setTimeout(() => recorder.stop(), 1500);
+        return promise;
+    }
+    
+    async function setVideo(t, obj)
+    {
+        const video = document.createElement('video');
+        video.muted = true;
+        video.srcObject = obj;
+        const watcher = new EventWatcher(t, video, [ 'loadedmetadata' ]);
+        await watcher.wait_for([ 'loadedmetadata' ]);
+        return video.videoHeight / video.videoWidth;
+    }
+
+    promise_test(async (t) => {
+        if (window.testRunner)
+            testRunner.setMockCameraOrientation(90);
+        const stream = await navigator.mediaDevices.getUserMedia({ video : true });
+        let result1 = await setVideo(t, stream);
```

Comment:
```suggestion
        const result1 = await setVideo(t, stream);
```
Ditto for result2 and blob.